### PR TITLE
graduate MatchLabelKeysInPodAffinity to beta

### PR DESCRIPTION
**Pull request by sanposhiho**
The doc update to graduate MatchLabelKeysInPodAffinity to beta.
Part of https://github.com/kubernetes/enhancements/issues/3633

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
LGTM for docs

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 1]
```diff
@@ -363,10 +363,10 @@ null `namespaceSelector` matches the namespace of the Pod where the rule is defi
 {{< feature-state for_k8s_version="v1.29" state="alpha" >}}
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:363]: This feature-state line also needs an update to show it's in the 'beta' stage for version 1.30. It'd be great if we could use `{{< feature-state feature_gate_name="MatchLabelKeysInPodAffinity >}}` but the improved shortcut functionality isn't synced with `dev-1.30` yet, a manual bump might be necessary for now.

```suggestion
{{< feature-state for_k8s_version="v1.30" state="beta" >}}
```

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 2]
```diff
@@ -413,10 +413,10 @@ spec:
...
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:413]: Similarly, for this line, an update to the feature-state is needed.
```suggestion
{{< feature-state for_k8s_version="v1.30" state="beta" >}}
```

**Comment by sanposhiho** [comment]
@dipesh-rawat Thanks, updated.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/match-label-keys-in-pod-affinity.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.29"
+    toVersion: "1.29"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
 ---
 Enable the `matchLabelKeys` and `mismatchLabelKeys` field for
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/match-label-keys-in-pod-affinity.md:17]: For beta (and for GA), we prefer accurate grammar.

```suggestion
Enable the `matchLabelKeys` and `mismatchLabelKeys` fields for
```

**Comment by Vyom-Yadav** [comment]
Hey @sanposhiho :wave: please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

---

This is already under review so just a friendly reminder :)

**Comment by sanposhiho** [comment]
Unfortunately we couldn't make it in the code freeze.

https://github.com/kubernetes/enhancements/issues/3633#issuecomment-1981634586

/close

**Comment by drewhagen** [comment]
Heads up that `dev-1.31` branch is created now, if you choose to reopen this for the new release.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/match-label-keys-in-pod-affinity.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.29"
+    toVersion: "1.29"
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/match-label-keys-in-pod-affinity.md:12]:
```suggestion
    toVersion: "1.30"
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates/match-label-keys-in-pod-affinity.md [hunk 5]
```diff
@@ -9,6 +9,10 @@ stages:
...
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/match-label-keys-in-pod-affinity.md:15]:
```suggestion
    fromVersion: "1.31"
```

**Review by Princesso — COMMENTED** [review]
/lgtm

**Comment by hacktivist123** [comment]
Hello @sanposhiho 👋! I'm reaching out from the Docs team. Just checking in as we approach Docs Freeze on Tuesday July 30th 18:00 PDT. This documentation appears to still be under review. To meet the Docs Freeze, this PR must have a technical review as well as lgtm and approve labels applied, without any unaddressed comments or concerns from SIG Docs. The status of this enhancement is marked as at risk for docs freeze. Thank you!

**Comment by sanposhiho** [comment]
/cc @Huang-Wei 

for the review from SIG.

Who should I ask for the approval from SIG-docs?

**Review by Huang-Wei — COMMENTED** [review]
/approve

**Review by Huang-Wei — COMMENTED** [review]
/approve

**Pull request merged**